book/languages/js/intro.md

```diff
@@ -0,0 +1,44 @@
+# Introduction
+
+## Inception
+
+**JavaScript (JS)**, but the official name is **ECMAScript**. <br> This language is really fast scripting language using specially for web pages, for **frontend** (Vue.js, React.js etc.) and for **backend** (node.js etc.). It can be used also for creating desktop and mobile native apps (Proton, Electron).
+**Brednan Eich** created that language, and it firstly appeared in **December 4, 1995**. JS became an **ECMA standard in 1997**. <br> File extension is **.js**
+
+## Trademark
+
+Actually Oracle have a trademark to JavaScript. Yes, these guys from Silicon Valley that are making virtual machines software.
+
+## Versions
+
+| **Year**      | **Name**      | **Description** |
+| ------------- | ------------- | ----- |
+| 1997          | ECMAScript 1  | First Edition |
+| 1998          | ECMAScript 2  | Editorial changes only |
+| 1999          | ECMAScript 3  | Added Regular Expressions, Added try/catch |
+|               | ECMAScript 4  | Was never released |
+| 2009          | ECMAScript 5  | Added "strict mode". Added JSON support. |
+| 2011          | ECMAScript 5.1| Editorial changes |
+| 2015          | ECMAScript 6  | Many new features (including let, const and default parameter values) |
+| 2016          | ECMAScript 7  | Added exponential operator (**), Added Array.prototype.includes  |
+| 2017          | ECMAScript 8  | Added string padding. Added new Object properies. Added Async functions. Added Shared Memory. |
+
```

Comment:
Not sure if there should be a Version table, maybe there should be instead a text About the changes and also it shouldn't get to technical